Source/Core/DiscIO/FileBlob.cpp

```diff
@@ -93,23 +93,16 @@ bool ConvertToPlain(BlobReader* infile, const std::string& infile_path,
       success = false;
       break;
     }
-    if (!outfile.WriteBytes(buffer.data(), sz))
+    if (!tempfile.WriteBytes(buffer.data(), sz))
     {
       PanicAlertFmtT("Failed to write the output file \"{0}\".\n"
                      "Check that you have enough space available on the target drive.",
-                     outfile_path);
+                     tempfile_path);
       success = false;
       break;
     }
   }
 
-  if (!success)
-  {
-    // Remove the incomplete output file.
-    outfile.Close();
-    File::Delete(outfile_path);
-  }
-
```

Comment:
If you want to get rid of this, you'll have to update the Android code in addition to the DolphinQt code.

Reply:
That's a good point. I only removed it because QTemporaryFile handles that automatically. I'll probably just restore this for now since it shouldn't make any difference for the Qt code.